frontend/packages/console-shared/src/components/pod/PodRing.tsx

```diff
@@ -62,8 +62,8 @@ const PodRing: React.FC<PodRingProps> = ({
     handleScaling(clickCount + operation);
   };
 
-  const isKnative = _.get(obj, 'metadata.ownerReferences[0].kind') === 'Revision';
-  const isScalingAllowed = !isKnative && editable && enableScaling;
+  const isKnativeRevision = obj.kind === 'Revision';
+  const isScalingAllowed = !isKnativeRevision && editable && enableScaling;
```

Comment:
This needs a tech debt item to avoid bringing in knativeisms into the shared code. I'm not sure how else to solve this right now though.
We probably need to create some extensions.